themes/vuetober/src/store/index.js

```diff
@@ -40,6 +40,11 @@ export default new Vuex.Store({
     selectTeacher(state, teacher) {
       state.selectedTeacher = teacher;
     },
+    addFeedback(state, feedback) {
```

Comment:
Toto sa mi nezda, spolieha sa to na to, ze IDcka su nejako incrementovane po jedno a zoradene, to ale vobec nemusi platit vzdy. Neda sa na to spolahnut.

Co chces docielit touto metodou?

Reply:
Ja hned po odoslani feedbacku pushujem usera na stranku ucitela ktory fetchuje vsetky data ucitelov. A ono akokeby fetchnem tie feedbacky skorej ako sa odosle ten novy takze ten novy neni zobrazeny medzi ostatnymi

Reply:
Mne to updatne teacherov vseobecne, ale selectnuty teacher ostane nezmeneny. Neviem ako to inac docielit.

Reply:
Nejako som to vyriesil. Asi sa to da aj bez toho fetchu navyse ale inac mi to neslo